math/geometry/straightlines.go

```diff
@@ -27,27 +28,27 @@ func Section(p1, p2 *Point, r float64) Point {
 	return point
 }
 
-// Calculates the slope (gradient) of a line.
+// Slope Calculates the slope (gradient) of a line.
 func Slope(l *Line) float64 {
 	return (l.P2.Y - l.P1.Y) / (l.P2.X - l.P1.X)
 }
 
-// Calculates the Y-Intercept of a line from a specific Point.
+// Intercept Calculates the Y-Intercept of a line from a specific Point.
 func Intercept(p *Point, slope float64) float64 {
```

Comment:
Can you call this `YIntercept` instead of just `Intercept`? Because we can also have `XIntercept`s and this would be confusing.

Reply:
Done